### PR TITLE
update readme to reflect latest version of images

### PR DESCRIPTION
**Pull request by bendavies**
# Usabilla PHP Docker Template

Reviewers: @usabilla/oss-docker

## Type

Please specify the type of changes being proposed:

| Q                 | A
| ----------------- | -----
| Documentation?    | yes
| Dockerfile change?| no
| Build feature?    | no
| Apply CVE Patch?  | no
| Remove CVE Patch? | no

## Changelog

- Please make sure you have checked our [Contributing guidelines](https://github.com/usabilla/php-docker-template/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ All images are based on their official variants, being:
 ## Alpine Linux situation
 
 Even though both of the images are based on the Alpine Linux, the PHP official repository gives us the option to choose
-between its versions, at this moment being `3.8` or `3.9`.
+between its versions, at this moment being `3.7`, `3.8`, `3.9` or `3.10`.
```
Comment (bendavies) [inline README.md]: i didn't really understand this sentence. have i corrected it properly?

File: README.md [hunk 2]
```diff
@@ -89,9 +89,9 @@ In order to provide upgrade path we intend to keep one or more versions of PHP a
 The tag naming strategy consists of (Read as a regex):
 
 - PHP: `(phpMajor).(phpMinor)-(cli|fpm)-(alpine|future supported OSes)(alpineMajor).(alpineMinor)(-dev)?`
-  - Example: `7.2-fpm-alpine3.8`, `7.2-fpm-alpine3.8-dev`
+  - Example: `7.3-fpm-alpine3.10`, `7.3-fpm-alpine3.10-dev`
   - Note: The minor version might come followed by special versioning constraints in case of betas, etc. For instance
-   `7.3-rc-fpm-alpine3.8-dev`
+   `7.4-rc-fpm-alpine3.10-dev`
```
Comment (renatomefi) [inline README.md]: We're not building the 7.4-rc, do you want to add it to the matrix?
Let's hope the tests will pass!

File: README.md (hunk 1 above)
Comment (renatomefi) [reply README.md]: Yes!
The intent is to say you can choose between alpine versions

Comment (cvmiert) [reply README.md]: According to the [current documentation](https://github.com/docker-library/docs/blob/master/php/README.md#supported-tags-and-respective-dockerfile-links), `3.7` and `3.8` are no longer supported. Can we drop them as well?

File: README.md (hunk 2 above)
Comment (bendavies) [reply README.md]: sure.

**Comment by bendavies** [comment]
Are we waiting for anything here?

**Comment by renatomefi** [comment]
Hey @bendavies, since we cannot build 7.4 yet if you could please remove it from the docs!
Also as @cvmiert said, let's not list alpine 3.7 and 3.8 any longer!

That should be it, thanks!

**Comment by bendavies** [comment]
@renatomefi done

**Pull request merged**